modules/magnolia/src/main/scala/pureconfig/module/magnolia/semiauto/writer.scala

```diff
@@ -1,13 +1,13 @@
 package pureconfig.module.magnolia.semiauto
 
-import scala.language.experimental.macros
-import scala.reflect.ClassTag
-
 import magnolia._
 import pureconfig.ConfigWriter
 import pureconfig.generic.{ CoproductHint, ProductHint }
 import pureconfig.module.magnolia.MagnoliaConfigWriter
 
+import scala.language.experimental.macros
```

Comment:
nit: can we revert all import changes in files that you didn't touch? This tends to pollute the commit log for a file and makes it harder to understand what really changed at a glance.

Reply:
@ruippeixotog @jcazevedo I removed import changes and unnecessary diffs (adopting scalafmt may help ensure this doesn't happen again... actually I assumed Intellij's formatting changes were unimportant _because_ scalafmt wasn't enforcing the style, Sorry! 😓)